src/sync/incomingSyncUtils.js

```diff
@@ -632,6 +640,27 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
       numberSequence.addNumberToReuse(numberToReuse);
       break;
     }
+    case 'Pref': {
+      const { item } = record;
+      if (item === 'store_preferences') {
+        try {
+          const recordData = JSON.parse(record?.data);
+          const recordKeys = Object.keys(recordData);
+          recordKeys.forEach(key => {
```

Comment:
Not required change (stylistic preference), but whenever I need to iterate through key values:
Object.entries(recordDate).forEach(([key, value]) => {
}

Reply:
Yeah, much nicer. Can't think why I went with this option! 27d82b07c910015038d36b761ee6136d127fc75c.